packages/veritone-json-schemas/schemas/vtn-standard/transcript/transcript.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://docs.veritone.com/schemas/vtn-standard/transcript.json",
+  "title": "vtn-standard.object",
+  "description": "Standard engine output for Object at Veritone",
+  "type": "object",
+  "definitions": {
+    "words": {
+      "type": "array",
+      "minItems": 1,
+      "if": {
+        "minItems": 2
+      },
+      "then": {
+        "items": {
+          "type": "object",
+          "properties": {
+            "word": {
+              "$ref": "master.json#/definitions/word"
+            },
+            "confidence": {
+              "$ref": "master.json#/definitions/confidence"
+            },
+            "utteranceLength": {
+              "$ref": "master.json#/definitions/utteranceLength"
+            },
+            "bestPath": {
+              "type": "boolean"
+            }
+          },
+          "required": [
+            "word",
+            "confidence",
+            "utteranceLength"
+          ]
+        },
+        "uniqueItems": true,
+        "requireBestPath": true
+      },
+      "else": {
+        "items": {
+          "type": "object",
+          "properties": {
+            "word": {
+              "$ref": "master.json#/definitions/word"
+            },
+            "confidence": {
+              "$ref": "master.json#/definitions/confidence"
+            }
+          },
+          "required": [
+            "word"
+          ]
+        }
+      }
+    }
+  },
+  "properties": {
+    "validationContracts": {
+      "type": "array",
+      "items": {
+        "const": "transcript"
```

Comment:
I don't think this allows for other `validationContracts` values to be present in the array (which should be allowed for it to still validate as a transcript).

Reply:
This is on purpose for now, to ensure they are one to one and to make sure the correct value is provided.

My general plan for this is to create each sub schema one by one then merge them all into the master, otherwise if you develop on the master schema directly the matching for JSON schema makes it hard to accomplish, especially without breaking existing property matches.